rfc-editor/rfc8794.xml

```diff
@@ -1823,19 +1623,19 @@ then the first instance of the Identically Recurring Element should be used
 for interpretation.</t>
 </section>
 
-<section anchor="textual-expression-of-floats"><name>Textual Expression of Floats</name>
-<t>When a Float value is represented textually in an EBML Schema, such as
-within a default or range value, the Float values <bcp14>MUST</bcp14> be
+<section anchor="textual-expression-of-floats"><name>Textual expression of floats</name>
+<t>When a float value is represented textually in an EBML Schema, such as
+within a default or range value, the float values <bcp14>MUST</bcp14> be
 expressed as Hexadecimal Floating-Point Constants as defined in the C11
-standard <xref target="ISO.9899"></xref> (see Section 6.4.4.2 on Floating
+standard <xref target="ISO.9899"></xref> (see section 6.4.4.2 on Floating
 Constants). <xref target="tableFloatExamples"></xref> provides examples of
-expressions of Float ranges.</t>
-<table anchor="tableFloatExamples"><name>Example of Floating-Point Values and
-Ranges as Decimal and Hexadecimal Floating-Point Constants</name>
+expressions of float ranges.</t>
+<table anchor="tableFloatExamples"><name>Example of Floating-Point values and
+ranges as decimal and Hexadecimal Floating-Point Constants </name>
```

Comment:
Is there a reason that `Floating-Point` and `Floating-Point Constants` are still “capitalised”?

Reply:
I think it refers to `expressed as Hexadecimal Floating-Point Constants as defined in the C11 standard [@!ISO.9899]` which is considered a defined term.

Reply:
okay, thank you for the explanation